### ISSUE TITLE
No more att?

### ISSUE THREAD

**Issue by righidiego** [Lunat1q/Catchem-PoGo#2288]
This project is dead?

**Comment by Leg3ndAry** [comment]
Are you dumb? Read before writing!!! #2282

**Comment by DeCarvalho83** [comment]
yeah sure its dead, it doesnt matter if the guy came here saying its not dead cuz it is.. theres no news nor anything... 
They should buy the api from bossland and sell like pgbot .. would be really better ..

**Comment by Lunat1q** [comment]
No need to tell me what i SHOULD do.
@DeCarvalho83.
Project is frozen untill API became public again.
Selling bot is illegal.

**Comment by DeCarvalho83** [comment]
really that no one need to tell what u should do? i dont care about you i wasnt sending a msg to you.. so get back to ur dark room poser.

Good thing that we have pgbot to use :) for 1.85$/month

LoSER
